### ISSUE TITLE
Can have Contact list with Simprints and regular results

### ISSUE THREAD

**Issue by abbyad** [medic/medic-webapp#3537]
It is possible to have a contact list with Simprints and regular results, as seen here:
![image](https://user-images.githubusercontent.com/789512/26986766-f6cc7d68-4d16-11e7-98dc-b6b8b1bd6df9.png)

To reproduce:
- Do a Simprints search
- Select a person from the list
- Edit that person and save
- Press the back arrow in the header bar to bring you to LHS with list of people + Simprints results

**This is low priority, especially that the code behind this is likely to be replaced due to https://github.com/medic/medic-webapp/pull/3510#discussion_r119238149**

**Comment by nice-snek** [comment]
hi frand @estellecomment, please [triage](https://github.com/medic/medic-docs/blob/master/development/workflow.md#triaging-old-issues) before the end of this sprint.

**Comment by estellecomment** [comment]
@abbyad I don't know that feature well, so I don't understand what the issue is. What's the expected vs actual behavior?

**Comment by abbyad** [comment]
This should be revisited as part of AT for #3511

**Comment by alxndrsn** [comment]
#3511 is closed.  What's the status of this ticket?

**Comment by abbyad** [comment]
Back to blocked by #3511, will AT as soon as it is ready.

**Comment by abbyad** [comment]
Closing since I could not reproduce this particular issue with `2.14.0-beta.6`.
